scripts/pirateofdoom.py

```diff
@@ -43,9 +44,25 @@
     "printing": "scribblin'",
     "load": "haul",
     "connect to": "board",
+    "connecting": "boarding",
     "collects": "hoards",
     "prime tower": "buoy",
     "change log": "captain's log",
     "my": "me",
-    "removable drive": "life boat"
+    "removable drive": "life boat",
+    "print core": "scribbler",
+    "printcore": "scribbler",
+    "start": "Sail ho!",
```

Comment:
This one is a bit of a problem. We often have stuff like "unable to start a print job" in our translations. That would become "unable to sail ho! a print job". We never say "start" separately.